spec/unit/delta_tables_generator_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe DeltaTablesGenerator do
+  let(:db) { Sequel::Model.db }
+
+  describe '#generate' do
+    context 'on empty database' do
+      before do
+        db[:measures].delete
+        db[:goods_nomenclatures].delete
```

Comment:
Is the DB not empty on the spec runtime?

Reply:
It should, but I had some erratic failures in the past due to specs running in random order and db cleaner not working as I expected, so I just wanted to be extra sure here

Reply:
Yeah fair enough.